lib/matplotlib/axis.py

```diff
@@ -916,6 +916,12 @@ def set_tick_params(self, which='major', reset=False, **kwargs):
 
         For documentation of keyword arguments, see
         :meth:`matplotlib.axes.Axes.tick_params`.
+
+        See Also
+        --------
+        .Axis.get_tick_params
+            View the current style defaults for ticks, ticklabels, and
```

Comment:
Again, I don't think these are necessarily the defaults, are they?  They are the current settings...